src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/Responses/GetCohortResponse.cs

```diff
@@ -15,5 +15,6 @@ public sealed class GetCohortResponse
         public string LatestMessageCreatedByProvider { get; set; }
         public bool IsApprovedByEmployer { get; set; }
         public bool IsApprovedByProvider { get; set; }
+        public bool EmployerCanApprove { get; set; }
```

Comment:
I think I'd rather call this IsCompleteForEmployer, as the approve option can be withdrawn for a number of reasons (eg. cohort not with user, agreement not signed) and they don't all have the same effect on the markup produced, so it could be confusing to have something with a vague name.